code/modules/admin/verbs/toggle_hard_dels.dm

```diff
@@ -0,0 +1,18 @@
+/client/proc/hard_deletion_toggle()
+	set category = "Debug"
+	set name = "Enable/Disable Hard Deletes"
+
+	var/static/list/warned_users
+
+	LAZYINITLIST(warned_users)
```

Comment:
```suggestion
	var/static/list/warned_users = list()
```
This will be initialized a single time.

Reply:
initializing static vars happens at world load in global namespace as if they were global variables, its why you can't initialize it with vars from the object on first run.